internal/mock_api/authentication/authentication.go

```diff
@@ -24,7 +24,6 @@ type UserAuthentication struct {
 
 func AuthenticationMiddleware(next mock_api.MockEndpoint) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		w.Header().Set("Content-Type", "application/json")
```

Comment:
Dumb Q: How is removing this header solving the issue? Is it set elsewhere?

Reply:
Yep: https://github.com/twitchdev/twitch-cli/pull/86/files#diff-88d80dd9c5f8665fa78df1e7795229c3e309abf9e4889eecaaf6c5b1ce68a496R103